src/common/file_util.cpp

```diff
@@ -304,20 +304,41 @@ bool DeleteDir(const std::string& filename) {
     return false;
 }
 
-bool Rename(const std::string& srcFilename, const std::string& destFilename) {
-    LOG_TRACE(Common_Filesystem, "{} --> {}", srcFilename, destFilename);
+bool Rename(const std::string& srcFilePath, const std::string& destFileName) {
+    LOG_TRACE(Common_Filesystem, "{} --> {}", srcFilePath, destFileName);
 #ifdef _WIN32
-    if (_wrename(Common::UTF8ToUTF16W(srcFilename).c_str(),
-                 Common::UTF8ToUTF16W(destFilename).c_str()) == 0)
+    if (_wrename(Common::UTF8ToUTF16W(srcFilePath).c_str(),
+                 Common::UTF8ToUTF16W(std::string(GetFilename(destFileName))).c_str()) == 0)
```

Comment:
This changes the behaviour of the Rename function, which may break existing emulated applications using `FSUSER::RenameFile` or `FSUSER::RenameDirectory`. A homebrew app must be created that calls this API and verifies its behaviour.

Reply:
In theory, this should behave the same as it did before on Android, so if this breaks something, it would have always been broken on Android. Having some way to check would be helpful though.